R/commonMachineLearningRegression.R

```diff
@@ -20,14 +20,29 @@
   return(colors)
 }
 
+.mlSetSeed <- function(options) {
```

Comment:
isn't it possible to use `.setSeedJASP`? See https://github.com/jasp-stats/jaspBase/blob/b2cdc13c86a0233d0531b8232b921c7aa754dc26/R/common.R#L627 Doesn't really matter but perhaps there is a reason?

Reply:
You are right, we can use that